source/documentation/other-topics/custom-default-backend.md

```diff
@@ -0,0 +1,78 @@
+
+```
+$ kubectl -n ${namespace} create -f custom-default-backend.yaml
+service "nginx-errors" created
+deployment.apps "nginx-errors" created
+```
+
+This should have created a Deployment and a Service with the name nginx-errors.
+
+```
+$ kubectl -n ${namespace} get deploy,svc
+NAME                           DESIRED   CURRENT   READY     AGE
+deployment.apps/nginx-errors   1         1         1         10s
+
+NAME                   TYPE        CLUSTER-IP  EXTERNAL-IP   PORT(S)   AGE
+service/nginx-errors   ClusterIP   10.0.0.12   <none>        80/TCP    10s
+```
+
+##### Defining annotations in Ingress file.
+
+Final step is to to use an [Default Backend][default-backend-annotation] annotation in your Ingress, This annotation is of the form nginx.ingress.kubernetes.io/default-backend: <svc name> to specify a custom default backend. This <svc name> is a reference to a service inside of the same namespace in which you are applying this annotation. This annotation overrides the global default backend.
+
+This service will handle the response when the service in the Ingress rule does not have active endpoints. If a default backend annotation is specified on the ingress, the errors will be routed to that annotation's default backend service (instead of the global default backend).
+
+Example Usage:
+```
+annotations:
+  # This is referencing the SAME NAMESPACE that this resource is in
+  nginx.ingress.kubernetes.io/default-backend: nginx-errors
+```
+
+This default-backend service also handle the error responses if both [default-backend][default-backend-annotation] annotation and the [custom-http-errors][custom-http-error-annotation] annotation is set. To configure which HTTP status codes Nginx should be forwarding to the default-backend using the `custom-http-errors` property, which can be configured using the [default-backend][default-backend-annotation] annotation in your Ingress.
+
+Custom HTTP Errors annotation will set NGINX proxy-intercept-errors to your custom default backend associated with this ingress in your namespace. If custom-http-errors is also specified globally, the error values specified in this annotation along with custom default backend will override the global value for the given ingress' hostname and path.
+
+Example Usage:
+
+```
+annotations:
+  nginx.ingress.kubernetes.io/default-backend: nginx-errors
+  nginx.ingress.kubernetes.io/custom-http-errors: "404,415"
+```
+
+#### Use platform-level error page
+
+Some teams want application's serve their own error's `example 404's`, but want to serve cloud platforms custom error page from default backend at ingress controller for timeouts errors like 502,503 and 504, this can be done by using [custom-http-errors][custom-http-error-annotation] annotation in your ingress for error codes teams want to serve the cloud platforms custom error page.
+
+Example Usage:
+
+```
+annotations:
```

Comment:
Can we have a more complete example (or a link to one) so that users know where, and in which file, this config should go?

Reply:
Added an example yaml file